lib/ui/widgets/custom_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class CustomButton extends StatelessWidget {
```

Comment:
**suggestion:** Move the Styling to the theme then this component becomes obsolete

Reply:
done